views/partials/related/more-ons.html

```diff
@@ -15,7 +15,8 @@ <h2 class="more-ons__type-text c-box__title-text">Latest {{preposition}}</h2>
 						<li class="more-ons__story">
 							{{#if @first}}{{#if image}}
 							<div class="more-ons__image">
-								{{#>link href=url type='more-ons-image'}}
+								{{! As they are duplicate links, hide the links from assistive technologies and prevent them from being tabbed to. }}
```

Comment:
**Suggestion:** I might be over thinking this but as this comment slightly aims to educate people as to why we want to do this it might also be nice to make it clear how thats being achieved?

```suggestion
								{{! As they are duplicate links, hide the links (ariaHidden) from assistive technologies and prevent them from being tabbed to (tabindex). }}
```